x-pack/legacy/plugins/watcher/common/constants/time_units.ts

```diff
@@ -5,7 +5,6 @@
  */
 
 export const TIME_UNITS: { [key: string]: string } = {
-  MILLISECOND: 'ms',
```

Comment:
`ms` was originally added for simulating a watch, but later found out it was not a valid time unit and forgot to remove from this file.